frontend/src/metabase/writeback/components/ActionCreator/FormCreator/EmptyFormPlaceholder.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { t } from "ttag";
+
+import Icon from "metabase/components/Icon";
+
+import {
+  EmptyFormPlaceholderWrapper,
+  ExplainerText,
+  ExampleButton,
+  IconContainer,
+  TopRightIcon,
+} from "./FormCreator.styled";
+
+export const EmptyFormPlaceholder = ({
+  onExampleClick,
+}: {
+  onExampleClick: () => void;
+}) => (
+  <EmptyFormPlaceholderWrapper>
+    <IconContainer className="text-brand p2 relative inline-block">
+      <Icon name="sql" size={62} />
+      <TopRightIcon name="insight" size={24} />
+    </IconContainer>
```

Comment:
Can we avoid CSS class usage here?

Reply:
🤦  thanks, this is actually already in emotion, I just didn't remove the classes.